dashboard/dashboard-user.md

```diff
@@ -0,0 +1,81 @@
+---
+title: TiDB Dashboard User Management
+summary: Learn how to create SQL users to access TiDB Dashboard.
+aliases: ['/docs/dev/dashboard/dashboard-user/']
+---
+
+# TiDB Dashboard User Management
+
+TiDB Dashboard uses the same user privilege system and sign-in authentication as TiDB. You can control and manage TiDB SQL users to limit their access to TiDB Dashboard. This document describes the least privileges required for TiDB SQL users to access TiDB Dashboard and exemplifies how to create a least-privileged SQL user.
+
+For details about how to control and manage TiDB SQL users, see [TiDB User Account Management](/user-account-management.md).
+
+## Description to required privileges
+
+To access TiDB Dashboard when [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is not enabled on the connected TiDB server, the SQL user should have **all** the following privileges:
+
+- PROCESS
+- SHOW DATABASES
+- CONFIG
+- DASHBOARD_CLIENT
+
+To access TiDB Dashboard when [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security) is enabled on the connected TiDB server, the SQL user should have **all** the following privileges:
+
+- PROCESS
+- SHOW DATABASES
+- CONFIG
+- DASHBOARD_CLIENT
+- RESTRICTED_TABLES_ADMIN
+- RESTRICTED_STATUS_ADMIN
+- RESTRICTED_VARIABLES_ADMIN
+
+To modify the configuration items on the interface after signing in with TiDB Dashboard, the SQL user must also have the following privilege:
+
+- SYSTEM_VARIABLES_ADMIN
+
+> **Note:**
+>
+> Users with high privileges such as `ALL PRIVILEGES` or `SUPER` can still sign in with TiDB Dashboard. Therefore, to comply with the least privilege principle, it is highly recommended that you create users with the preceding specific privileges to prevent unintended operations. See [Privilege Management](/privilege-management.md) for more information on these privileges.
+
+When the SQL user does not meet the preceding privilege requirements, it fails to sign in with TiDB Dashboard, as shown below.
+
+![insufficient-privileges](/media/dashboard/dashboard-user-insufficient-privileges.png)
+
+## Example: Create a least-privileged SQL user to access TiDB Dashboard
```

Comment:
It seems this part content format is not exactly the same as the Chinese version.